<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMvColorImagesCut</class>
 <widget class="QWidget" name="CMvColorImagesCut">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>892</width>
    <height>652</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>图片裁剪</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QGraphicsView" name="graphicsView_mainImage">
       <property name="minimumSize">
        <size>
         <width>500</width>
         <height>600</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>42</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>42</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>2</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="pushButton_zoomIn">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>放大</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_zoomOut">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>缩小</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_zoomBest">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>最好尺寸</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_lockRoiMove">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>锁定ROI</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>63</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_mousePos">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="minimumSize">
        <size>
         <width>382</width>
         <height>602</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>380</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>输入设置</string>
        </attribute>
        <widget class="QLabel" name="label_funcName_2">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>30</y>
           <width>81</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>检测器名称：</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_roiCreat_2">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>70</y>
           <width>91</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>ROI来源：</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkBox_enableFunc_2">
         <property name="geometry">
          <rect>
           <x>270</x>
           <y>20</y>
           <width>101</width>
           <height>31</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>101</width>
           <height>31</height>
          </size>
         </property>
         <property name="text">
          <string>启用检测器</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkBox_roiCreat_2">
         <property name="geometry">
          <rect>
           <x>270</x>
           <y>60</y>
           <width>101</width>
           <height>31</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>101</width>
           <height>31</height>
          </size>
         </property>
         <property name="text">
          <string>自己创建</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <property name="tristate">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QPlainTextEdit" name="plainTextEdit_funcName_2">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>20</y>
           <width>131</width>
           <height>31</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>131</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>131</width>
           <height>31</height>
          </size>
         </property>
         <property name="overwriteMode">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox_imageSource_3">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>60</y>
           <width>131</width>
           <height>31</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>131</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>131</width>
           <height>31</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>正方形</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>圆形</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>多边形</string>
          </property>
         </item>
        </widget>
        <widget class="QTableWidget" name="tableWidget_input">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>120</y>
           <width>341</width>
           <height>361</height>
          </rect>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="horizontalScrollMode">
          <enum>QAbstractItemView::ScrollPerItem</enum>
         </property>
         <property name="showGrid">
          <bool>false</bool>
         </property>
         <property name="sortingEnabled">
          <bool>false</bool>
         </property>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderCascadingSectionResizes">
          <bool>false</bool>
         </attribute>
         <row>
          <property name="text">
           <string>1</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>2</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string>输入名称</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>数据来源</string>
          </property>
         </column>
         <item row="0" column="0">
          <property name="text">
           <string>彩色图像</string>
          </property>
         </item>
         <item row="0" column="1">
          <property name="text">
           <string>某个检测器的输出</string>
          </property>
         </item>
         <item row="1" column="0">
          <property name="text">
           <string>坐标系输入 </string>
          </property>
         </item>
         <item row="1" column="1">
          <property name="text">
           <string>某个检测器的输出</string>
          </property>
         </item>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_3">
        <attribute name="title">
         <string>掩膜设置</string>
        </attribute>
        <widget class="QWidget" name="widget_3" native="true">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>60</y>
           <width>371</width>
           <height>251</height>
          </rect>
         </property>
         <widget class="QLabel" name="label_penWidth_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>50</y>
            <width>111</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>画笔尺寸：</string>
          </property>
         </widget>
         <widget class="QSpinBox" name="spinBox_lightTime_3">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>50</y>
            <width>131</width>
            <height>31</height>
           </rect>
          </property>
          <property name="suffix">
           <string>像素</string>
          </property>
          <property name="minimum">
           <number>5</number>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_maskAll_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>160</y>
            <width>121</width>
            <height>40</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>掩盖所有</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_maskNull_2">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>110</y>
            <width>121</width>
            <height>40</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>清空掩膜</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_saveMask_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>210</y>
            <width>121</width>
            <height>40</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>保存修改</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_lightOutputIo_10">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>111</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>编辑方式：</string>
          </property>
         </widget>
         <widget class="QComboBox" name="comboBox_lightOutputIo_7">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>10</y>
            <width>131</width>
            <height>31</height>
           </rect>
          </property>
          <item>
           <property name="text">
            <string>增加掩膜</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>删除掩膜</string>
           </property>
          </item>
         </widget>
        </widget>
        <widget class="QLabel" name="label_dEtectionMask">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>111</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>检测区域的掩膜：</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox_dEtectionMask">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>20</y>
           <width>131</width>
           <height>31</height>
          </rect>
         </property>
         <item>
          <property name="text">
           <string>不使用掩膜</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>本检测器ROI内掩膜</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>其他算法的掩膜</string>
          </property>
         </item>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_4">
        <attribute name="title">
         <string>结果绘制</string>
        </attribute>
        <widget class="QCheckBox" name="checkBox_valuTrig_4">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>10</y>
           <width>80</width>
           <height>40</height>
          </rect>
         </property>
         <property name="text">
          <string>启用绘制</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="pushbutton_cuttingPicture">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>80</width>
           <height>40</height>
          </rect>
         </property>
         <property name="text">
          <string>显示裁剪图像</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_cuttingPicture">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>80</width>
           <height>40</height>
          </rect>
         </property>
         <property name="text">
          <string>裁剪图像：</string>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="leftMargin">
        <number>1</number>
       </property>
       <property name="topMargin">
        <number>1</number>
       </property>
       <property name="rightMargin">
        <number>1</number>
       </property>
       <property name="bottomMargin">
        <number>1</number>
       </property>
       <item>
        <widget class="QPushButton" name="pushButton_ones">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>单次</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_affirm">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>确定</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbPrev_cancel">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>取消</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="CMvFindRoundParamWidget.qrc"/>
 </resources>
 <connections/>
</ui>
